src/main/java/gregtech/api/recipes/logic/ParallelLogic.java

```diff
@@ -437,16 +437,19 @@ public static RecipeBuilder<?> doParallelRecipes(Recipe currentRecipe, RecipeMap
 
         // Simulate the merging of the maximum amount of recipes
         // and limit by the amount we can successfully merge
+        // Note that limitByOutput will be less than or equal to multiplierByInputs after this, so Math.min does not need using
```

Comment:
This comment makes sense only right now when we can check what Math.min call was removed.

Reply:
I slightly modified the first comment to better explain that this variable is also limited by our inputs.